azure-devops/azext_devops/dev/team/service_endpoint.py

```diff
@@ -112,8 +112,9 @@ def create_service_endpoint(service_endpoint_type, authorization_scheme, name,
             service_endpoint_authorization = EndpointAuthorization(
                 parameters={'accessToken': github_access_token},
                 scheme=SERVICE_ENDPOINT_AUTHORIZATION_PERSONAL_ACCESS_TOKEN)
+            service_endpoint_data = {'GeneratedBy': 'AzureDevopsExtensionFlow'}
```

Comment:
We should allow users to set this while creating SE. and if nothing is passed then we set the value here 
Also would be good to mention AzureDevops**CLI**ExtensionFlow
so that there is no collision in future